### PR TITLE
Update fedcloudclient command for the discovery of GPUs

### PR DESCRIPTION
**Pull request by sebastian-luna-valero**
# Summary

Thanks to @tdviet for sending the new command in today's Fedcloud Task Force meeting.

As discussed in https://github.com/EGI-Federation/documentation/pull/316#issuecomment-914414114 let's see what happens to line length.

### CODE REVIEW

**Comment by sebastian-luna-valero** [comment]
The error on line length keeps complaining. Any ideas @gwarf ?

File: content/en/users/cloud-compute/gpgpu/_index.md [hunk 1]
```diff
@@ -33,14 +33,13 @@ dashboards, [VMOps dashboard](../vmops), or [Infrastructure manager](../im).
 It is also possible to use the
 [fedcloudclient](https://fedcloudclient.fedcloud.eu/) for CLI access. Below is
 an example on how to use the `fedcloud` command to show the GPU properties of
-the available flavors:
+the available GPU flavors on all sites:
```
Comment (enolfc) [inline content/en/users/cloud-compute/gpgpu/_index.md:36]: this command is for a specific VO, may be worth clarifying that?

Comment (tdviet) [reply content/en/users/cloud-compute/gpgpu/_index.md:36]: Could it be "the available GPU flavors on all sites for the specific VO in the command:" ?

**Comment by gwarf** [comment]
As #318 didn't worked as expected I'm pushing some changes to your branch, to make some tests.

**Comment by gwarf** [comment]
Arf, now it's passing, but apparently without doing any real check :'(

**Comment by gwarf** [comment]
@sebastian-luna-valero can you please try to push an update to the markdown file (like by addressing the comment from Viet) to see if now it's properly linted and taking into account the exclusions?

File: content/en/users/cloud-compute/gpgpu/_index.md (hunk 1 above)
Comment (sebastian-luna-valero) [reply content/en/users/cloud-compute/gpgpu/_index.md]: Addressed in https://github.com/EGI-Federation/documentation/pull/317/commits/301e3eb254c514164b03087118e6c3b0749cfaab

Thanks both!

**Review by gwarf — APPROVED** [review]
LGTM.
Keeping the added changes on linting to avoid having yet another PR for this.

**Pull request merged**